include/range/v3/utility/polymorphic_cast.hpp

```diff
@@ -0,0 +1,24 @@
+//  (C) Copyright Kevlin Henney and Dave Abrahams 1999.
+//  Distributed under the Boost
+//  Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef RANGES_V3_UTILITY_POLYMORPHIC_CAST_HPP
+#define RANGES_V3_UTILITY_POLYMORPHIC_CAST_HPP
+
+#include <typeinfo>
```

Comment:
`<typeinfo>` isn't needed to do `dynamic_cast`s. Please remove this unnecessary #include.